frontend/src/container/GridGraphLayout/Graph/FullView/index.metricsBuilder.tsx

```diff
@@ -78,7 +80,7 @@ function FullView({
 
 	const isLoading = response.isLoading === true;
 
-	if (isLoading) {
+	if (isLoading || isDependedDataLoaded) {
```

Comment:
this or condition is not needed

Reply:
This is required as at the initial load it will show no data rather than loading.